### ISSUE TITLE
about the dataset process

### ISSUE THREAD

**Issue by GorgeousMe** [dongwei156/n2n#1]
can you provide the code of how to process the data? I find there exist some statistics problem about the cora dataset,Thank you

**Comment by dongwei156** [comment]
Hello, Dear GorgeousMe, Thank you very much for your invaluable comments. Our split ratio in Cora dataset is consistent with CS-GNN [ICLR2019]. The node id range of the training set is 0-139 and 640-1707; the node id range of the validation set is 140-639; finally, the node id range of the test set is 1708-2707. Thank you again for your suggestion.

**Comment by GorgeousMe** [comment]
Thanks for your carefully answers! But When I print the length of the three set, 1208 for train,500 for val,and 1000 for test. so I use the gcn data splits ,which cannot reproduct the result as the paper claimed.
